src/server/impl2/ViewerClient2.cpp

```diff
@@ -100,6 +100,7 @@ void ViewerClient2::notifyNewStreamer( const StreamerClient2* streamer ) {
 
     m_clientStreamAdded = false;
     int iTry            = 0;
+    // Todo fix this
```

Comment:
Why is this todo for ?
It is not clear why it is wrong. Please add a description of the problem so contributors can understand what needs to be fixed.